.github/workflows/pr_portal-api.yml

```diff
@@ -189,7 +181,7 @@ jobs:
       # https://github.com/cypress-io/github-action/issues/48
       fail-fast: false
       matrix:
-        machines: [ 1, 2, 3, 4, 5, 6, 7 ]
+        machines: [ 1, 2, 3, 4 ]
```

Comment:
7 machines would work fine if only 1 PR is executing the tests. But if there is more than 1, then those PRs would take longer to run their tests because it needs to wait for the other PRs to finish. 
This is because we have a limited number of ubuntu-latest machines we can connect to

Reply:
Cool